hack/update-bindata.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+# Copyright 2016 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o pipefail
+set -x
+
+if [ -z $KUBE_ROOT ]; then
+	echo "KUBE_ROOT not detected, setting default."
+	KUBE_ROOT="../../../"
+fi
+
+if [[ ! -d "${KUBE_ROOT}/examples" ]]; then
+	echo "This is meant to be run from the test/e2e/ directory by test/e2e/e2e.go or similar."
```

Comment:
maybe also echo that "${KUBE_ROOT}/examples" doesn't exist?

that way if that directory gets deleted/moved it's more obvious why the build is failing.